src/amp-story-player/amp-story-player-impl.js

```diff
@@ -1591,6 +1595,19 @@ export class AmpStoryPlayer {
     return index >= this.stories_.length || index < 0;
   }
 
+  /** @private */
+  initializeAutoplay_() {
+    if (!this.playerConfig_) {
+      return;
+    }
+
+    const {behavior} = this.playerConfig_;
+
+    if (behavior && behavior.autoplay === 'false') {
+      this.autoplay_ = false;
+    }
```

Comment:
Why not `this.autoplay_ = behavior && behavior.autoplay === "true"`? Usually setting booleans inside if statements can be simplified